src/app/public/modules/lookup/lookup.component.html

```diff
@@ -56,9 +57,26 @@
             (blur)="onAutocompleteBlur()"
             (keydown)="inputKeydown($event, $event.target.value)"
             (keyup)="inputKeyup($event)"
-            #lookupInput
           ></textarea>
         </sky-tokens>
+        <div *ngIf="selectMode === 'single'"
+          class="sky-lookup-single-control"
+        >
+          <textarea
+            class="sky-form-control sky-lookup-input"
+            rows="1"
+            skyAutocomplete
+            wrap="off"
+            [attr.aria-label]="ariaLabel"
+            [attr.aria-labelledby]="ariaLabelledBy"
+            [attr.placeholder]="placeholderText"
+            [autocompleteAttribute]="autocompleteAttribute"
+            [disabled]="disabled"
+            (blur)="onAutocompleteBlur()"
+            (keydown)="inputKeydown($event, $event.target.value)"
+            (keyup)="inputKeyup($event)"
```

Comment:
In the `inputKeydown()` and `inputKeyup()` methods, we're first checking if `selectMode !== 'single'`. Could we either drop that or just remove these listeners since we have a different input in the template for single select?

Reply:
Good catch - I just didn't catch that when I made the changes to stop the tokens from being in the DOM. I'd like to leave the if statement in the methods themselves just as a sanity check. However, I have removed the template calls to the methods and added a comment on both methods that the check is just a sanity check.